### PR TITLE
remove broken support links

### PR DESCRIPTION
**Pull request by wandechris**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,9 +3,9 @@
 This is the second release of the SpatialKey DM API command line tool. The initial release can be found at [here]https://github.com/SpatialKey/skdm/tree/v1
 
 # Introduction
-The [SpatialKey Data Management REST API](http://support.spatialkey.com/dmapi) allows developers to programmatically create and update CSV data or Shapefiles in SpatialKey without having to login.  With the Command Line Tool, typical users of SpatialKey can leverage this functionality without any programming knowledge.  You just need to learn a few of the basics and you will be all set to kicking off import and update jobs outside of SpatialKey.
+The [SpatialKey Data Management REST API] allows developers to programmatically create and update CSV data or Shapefiles in SpatialKey without having to login.  With the Command Line Tool, typical users of SpatialKey can leverage this functionality without any programming knowledge.  You just need to learn a few of the basics and you will be all set to kicking off import and update jobs outside of SpatialKey.
```
Comment (kevinkorngut) [inline README.md]: A couple of things
- did you intend to leave the brackets around SpatialKey Data Management REST API?
- and in the last sentence you say "outside of SpatialKey" while earlier you say "in SpatialKey"

Comment (rstehwien) [reply README.md]: @kevinkorngut @wandechris The brackets were part of markdown link syntax... with no link, they aren't needed anymore.

**Review by rstehwien — APPROVED** [review]
See change suggestion about []

File: README.md (hunk 1 above)
Comment (wandechris) [reply README.md]: @kevinkorngut it meant kicking off imports and updating jobs outside of the spatialkey app

Comment (wandechris) [reply README.md]: @rstehwien @kevinkorngut updated

Comment (brandonpurcell) [reply README.md]: Thanks, this looks good. I have a number of changes I need to make to the example data to rework it in the near future and will kick off a review.

**Pull request merged**